### DISCUSSION TITLE
Submission error with tensor shape misalignment

### DISCUSSION THREAD

**Discussion by bqpeng-1006** [PaddlePaddle/PaddleSpatial#115] [BaiduKDDCup2022] [answered]
Hi there,

I got an error like 

> The size of tensor a (288) must match the size of tensor b (432) at non-singleton dimension 1

in submission. However, I ran into no errors in my local evaluation process with the latest evaluation.py. Is it possible to provide the online test log to debug this error? Thank you.
Team name: explorer
User name: test_123456

Best,

**Comment by ramber1836** [comment, marked as answer]
More details: 

File "/tmp/tmpfk9p6u12/tcn/layers/Embed.py", line 118, in forward
RuntimeError: The size of tensor a (288) must match the size of tensor b (432) at non-singleton dimension 1

BTW, did you test your code with the toy test data?
